matrix/all-in-one.yaml

```diff
@@ -0,0 +1,575 @@
+  - name: OTEL_DOTNET_AUTO_TRACES_ADDITIONAL_LEGACY_SOURCES
+    description: Comma-separated list of additional legacy source names to be added to the tracer at the startup. Use it to capture `System.Diagnostics.Activity` objects created without using the `System.Diagnostics.ActivitySource` API.
+    default: ''
+    type: string
+    category: instrumentation
+  - name: OTEL_LOG_LEVEL
+    description: Sets the logging level for instrumentation log messages. Possible values are `none`, `error`, `warn`, `info`, and `debug`. The default value is `info`. Can't be set using the web.config or app.config files.
+    default: 'info'
+    type: enum
+    category: diagnostic logging
+    to_do_comment: handle enum values/mark as string
+  - name: OTEL_DOTNET_AUTO_LOG_DIRECTORY
+    description: Directory of the .NET tracer logs. The default value is `/var/log/opentelemetry/dotnet` for Linux, and `%ProgramData%\OpenTelemetry .NET AutoInstrumentation\logs` for Windows. Can't be set using the web.config or app.config files.
+    default: 'see descruotuin'
+    type: string
+    category: diagnostic logging
+    to_do_comment: defaultvalue
+  - name: OTEL_DOTNET_AUTO_TRACES_CONSOLE_EXPORTER_ENABLED
+    description: Whether the traces console exporter is activated. The default value is `false`.
+    default: 'false'
+    type: boolean
+    category: diagnostic logging
+  - name: OTEL_DOTNET_AUTO_METRICS_CONSOLE_EXPORTER_ENABLED
+    description: Whether the metrics console exporter is activated. The default value is `false`.
+    default: 'false'
+    type: boolean
+    category: diagnostic logging
+  - name: OTEL_DOTNET_AUTO_LOGS_CONSOLE_EXPORTER_ENABLED
+    description: Whether the logs console exporter is activated. The default value is `false`.The default value is `false`.
+    default: 'false'
+    type: boolean
+    category: diagnostic logging
+  - name: OTEL_DOTNET_AUTO_LOGS_INCLUDE_FORMATTED_MESSAGE
+    description: Whether the log state have to be formatted. The default value is `false`.
+    default: 'false'
+    type: boolean
+    category: diagnostic logging
+  - name: MANUAL SETTINGS
+    description: How to handle https://docs.splunk.com/observability/en/gdi/get-data-in/application/otel-dotnet/configuration/advanced-dotnet-configuration.html#environment-variables-for-manual-installation?
+    default: 'N/A'
+    type: N/A
+    category: manual
+    to_do_comment: See description
+instrumentations:
+  - keys:
+      - ASPNET
+    instrumented_libraries:
+      - name: ASP.NET Framework (.NET Framework)
+        supported_versions: See :ref:`dotnet-otel-versions`
+    comment: MVC / WebApi (Only integrated pipeline mode supported). Metrics requires trace instrumentation.
+    stability: beta
+    supportability_level: community
+    dependencies:
+      - name: ASP.NET Instrumentation for OpenTelemetry
+        link: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.Instrumentation.AspNet
```

Comment:
Maybe we could change it to package hyperlink e.g. `package-href`. If we would like to add hyperlink to source code as well I would rename `link` to `source-href`.

```suggestion
        package-href: https://www.nuget.org/packages/OpenTelemetry.Instrumentation.AspNet/1.6.0-beta.1
        source-href: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.Instrumentation.AspNet
```

We could also have `doc-href` if needed.

Some comment applies to other places.

Reply:
Depends on how we want this to be used. If it's useful for machines to parse this file to retrieve packages, then I guess that's a good idea; for documentation, I'd rather link to the item that users of OTel .NET will care the most about. For Go, I guess that'd be the module docs?

Reply:
For go it could be both "package path" and "package docs URL"
When rendering you could use the package path as "text" and use the "package docs URL" to provide an URL
Like we did here:  https://docs.splunk.com/observability/en/gdi/get-data-in/application/go/go-otel-requirements.html

Reply:
Makes sense. If we codify this in the gdi-specs, then the usage of one of the fields would be mandatory, but the usage of both could be optional, depending on what makes sense in the ecosystem.

Reply:
It is not always possible, even for .NET, to have both package and source link, it should be optional field.

Reply:
12ce8cc8e2454821554cafc936865f8388e5fe5d
and 022e5a729d2f179883ae837ee956a41f0e7e1a99